packages/blocto/src/index.ts

```diff
@@ -28,9 +28,9 @@ function BloctoWallet(): WalletInit {
               return item.id === (params && params[0] && params[0].chainId)
             })
             if (!chain) throw new Error('chain must be set before switching')
-            const providerRpcurl =
-              bloctoProvider.switchableNetwork[chain.id] &&
-              bloctoProvider.switchableNetwork[chain.id].rpc_url
+            const providerRpcurl = bloctoProvider.rpc
+              ? bloctoProvider.rpc
+              : undefined
```

Comment:
```suggestion
            const providerRpcurl = bloctoProvider?.rpc
```

Reply:
Can we also fix the casing on this var to `providerRpcUrl`?

Reply:
sure, fixed in [fa5edc6](https://github.com/blocknative/web3-onboard/pull/2082/commits/fa5edc67f6cc3a7ac1290545e2c23889ceb30572) and  aedd7d5